images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -24,13 +24,13 @@ restore_pv_data
 cd ${APP_ROOT}
 bin/rake evm:deployment_status
 
+# Generate httpd certificate
+/usr/bin/generate_miq_server_cert.sh
```

Comment:
Isn't this change making us regenerate the cert when it isn't necessary? Now we'll generate a cert even on straight redeploy ... I'm not sure what this commit it doing for this PR.